indy_node/test/request_handlers/conftest.py

```diff
@@ -43,6 +49,60 @@ def schema_request():
                               }})
 
 
+@pytest.fixture(scope="module")
+def rs_schema_handler(db_manager, write_auth_req_validator):
+    return RsSchemaHandler(db_manager, write_auth_req_validator)
+
+
+@pytest.fixture(scope="function")
+def rs_schema_request():
+    authors_did, name, version, type = "2hoqvcwupRTUNkXn6ArYzs", randomString(), "1.1", MARKER_RS_SCHEMA
```

Comment:
Since `type` is a language key word, it's better to use `_type` or something similar.

Reply:
I will update `type` to `_type`.